src/pages/SearchResults/SearchResults.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import { Heading } from 'react-aria-components';
+
+import { ListTitles } from './components/ListTitles/ListTitles';
+
+import { Header } from '@components/Header/Header';
+import { Sidenav } from '@components/Navigation/Sidenav';
+import { Search } from '@components/Searchbar/Search';
+import { ExtraFilters } from '@components/Searchbar/ExtraFilters';
+import { useQuery } from '@tanstack/react-query';
+import { TitleService } from '@services/Title/Title.service';
+import { Paragraph } from '@components/Paragraph/Paragraph';
+import { NextSeasons } from '@pages/Title/components/NextSeasons/NextSeasons';
+import { Recommendations } from '@pages/Title/components/Recommendations/Recommendations';
+import { NEXT_ANIME, RECOMMENDATIONS_ANIME } from '@/constants';
+
+export const SearchResults = () => {
+  const { data: newTitles } = useQuery({
+    queryKey: ['titles'],
+    queryFn: async () => {
+      return TitleService.getAllTitles();
+    },
+    staleTime: 1000 * 60 * 60 // 60 minutes caching
+  });
+  const [isOpenPopup, setIsOpenPopup] = React.useState<boolean>(false);
+
+  const togglePopup = (open: boolean) => {
+    setIsOpenPopup(open);
+  };
+  return (
+    <section className="search-results page-section">
+      <div className="home__container page-section__container">
+        <Header />
+        <Sidenav />
+
+        <main className="main">
+          <div className="search-results__container container">
+            <Search />
+
+            <button
+              className="search-block__button"
+              onClick={() => togglePopup(true)}
+            >
+              Додаткові фільтри
+            </button>
+            {isOpenPopup && (
+              <div className="search-block__filters">
+                <ExtraFilters togglePopup={togglePopup} />
+              </div>
+            )}
+
+            <Heading className="mb-[67px]">Результати пошуку... (12)</Heading>
+          </div>
+          <div className="container">
+            <ListTitles items={newTitles} />
+
+            <div className="search-results__footer ">
+              <div className="search-results__footer-col-1">
```

Comment:
you can remove this block for now

Reply:
what kind of block?

Reply:
sorry with this small preview it is not showed, so left it :)